DQMOffline/Trigger/python/ExoticaMonitoring_Client_cff.py

```diff
@@ -15,6 +15,37 @@
   
 )
 
+NoBPTXEfficiency = cms.EDAnalyzer("DQMGenericClient",
```

Comment:
@jalimena 
I think you need to update this :(

it should be
`NoBPTXEfficiency = cms.EDAnalyzer("DQMGenericClient",`
into
```
import FWCore.ParameterSet.Config as cms
from DQMServices.Core.DQMEDHarvester import DQMEDHarvester

NoBPTXEfficiency = DQMEDHarvester("MyHarvester",
```